Guided Hunting - Anomalous Office365 Exchange Sessions.ipynb

```diff
@@ -0,0 +1,547 @@
+{
```

Comment:
<p>Not an issue but with ws_config = WorkspaceConfig(workspace='Default') you don't need to specify 'Default' if you don't provide a workspace it will just use 'Default'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>